src/build/config/parse.h

```diff
@@ -74,7 +74,6 @@ typedef struct BldCfgCommand
 typedef struct BldCfgOptionGroup
 {
     const String *const name;                                       // Name
-    const unsigned int indexTotal;                                  // Total indexes for option
```

Comment:
It would be `name of option group` if anything, but we'd need to change a lot of other places as well. For me, it's pretty clear that name is for the option group, command, option, etc, but it we want to change it that should be in a separate commit.